doc/user/content/install.md

```diff
@@ -9,7 +9,7 @@ aliases:
  - /install
 ---
 
-Materialize was first built as a single binary that runs on a single node: `materialized`. To support mission-critical deployments at any scale, we are now evolving the binary into a [cloud-native platform](https://materialize.com/materialize-unbundled/) with built-in horizontal scaling, active replication and decoupled storage.
+Materialize was first built as a single binary that runs on a single node: `materialized`. To support mission-critical deployments at any scale, we are now evolving the binary into a [cloud-native platform](https://materialize.com/blog/materialize-unbundled/) with built-in horizontal scaling, active replication and decoupled storage.
```

Comment:
Should we link to the newest blogpost instead?

Reply:
Yes, good point! Just changed that!